pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -853,7 +857,9 @@ func (m *ManagerImpl) GetDeviceRunContainerOptions(pod *v1.Pod, container *v1.Co
 	}
 	if needsReAllocate {
 		klog.V(2).Infof("needs re-allocate device plugin resources for pod %s", podUID)
-		m.allocatePodResources(pod)
+		if err := m.allocatePodResources(pod); err != nil {
```

Comment:
Adding error checking here feels appropriate to me - the other usage of `allocatePodResources`, in the `managerImpl.Allocate` method performs error checking with the result of `allocatePodResources`.